cozepy/websockets/chat/__init__.py

```
@@ -130,10 +132,49 @@ class Data(BaseModel):
     data: Data
 
 
+# req
+class ConversationMessageCreateEvent(WebsocketsEvent):
+    """手动提交对话内容
+
+    若 role=user，提交事件后就会生成语音回复，适合如下的场景，比如帮我解析 xx 链接，帮我分析这个图片的内容等。若 role=assistant，提交事件后会加入到对话的上下文。
+    docs: https://www.coze.cn/open/docs/developer_guides/streaming_chat_event#46f6a7d0
+    """
+
+    class Data(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)